src/main/java/lotto/domain/lotto/LottoTicketBuyingRequest.java

```diff
@@ -0,0 +1,47 @@
+package lotto.domain.lotto;
+
+import lotto.domain.Money;
+
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+import static lotto.domain.lotto.LottoTicketGenerator.PRICE_EACH_LOTTO;
+
+
+public class LottoTicketBuyingRequest {
+    private static final String ERROR_MANUAL_LOTTO_PRICE_OVER = "[ERROR] 총 구입 금액보다 수동구입량이 많습니다.";
+
+    private final Money money;
+    private final LottoAmount manualLottoAmount;
+    private final List<LottoLine> manualLottoLineList;
```

Comment:
> LottoTickeyBuyingRequest 객체는 생성자 파라미터로 Money와 LottoAmount를 받아요. 그런 다음 생성자에서 유효성 검사를 한 뒤 수동 LottoLine 리스트를 만들어요.
생성자에서 리스트 생성만 해 놓고 이후에 데이터를 추가하는 것이 문제가 될 수 있을까요?

문제가 되지는 않습니다. 다만 다음과 같이 필드에서 바로 초기화해도 좋을 것 같아요 ㅎㅎ
생성자에서 하면 새로운 생성자가 추가되었거나 한 경우 누락될 여지도 있어서요 :)
```suggestion
    private final List<LottoLine> manualLottoLineList = new ArrayList();
```

Reply:
ed95e9b 이곳에 반영 하였습니다!